### PR TITLE
refactor: Design changes for reset icon in Ast-customizer

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Reset icon design changes

### Screenshots
- https://share.getcloudapp.com/12uo1Xe4

### Types of changes
- Non breaking changes

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/custom-controls/assets/css/unminified/color-rtl.css [hunk 1]
```diff
@@ -308,26 +308,31 @@
...
     top: 1.5px;
 }
-.ast-color-btn-reset-wrap .ast-reset-btn.components-button {
+.ast-color-btn-reset-wrap .ast-reset-btn.components-button, .ast-resp-slider-reset-wrap .ast-reset-btn.components-button {
 	color: #006BA1;
     height: 20px;
     width: 20px;
     padding: 4px;
-    box-shadow: inset 0 0 0 1px #007cba;
+	background: transparent;
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/color-rtl.css:316]: @imnavanath Check the alignment.

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css [hunk 2]
```diff
@@ -698,26 +698,31 @@
...
     top: 1.5px;
 }
-.ast-color-btn-reset-wrap .ast-reset-btn.components-button {
+.ast-color-btn-reset-wrap .ast-reset-btn.components-button, .ast-resp-slider-reset-wrap .ast-reset-btn.components-button {
 	color: #006BA1;
     height: 20px;
     width: 20px;
     padding: 4px;
-    box-shadow: inset 0 0 0 1px #007cba;
+	background: transparent;
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:706]: @imnavanath Check the alignment.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/customizer/custom-controls/assets/css/unminified/color-rtl.css (hunk 1 above)
Comment (imnavanath) [reply inc/customizer/custom-controls/assets/css/unminified/color-rtl.css:316]: Fixed, and it's aligned now - https://a.cl.ly/v1uNxPkx

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css (hunk 2 above)
Comment (imnavanath) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:731]: Fixed, it's aligned now - https://a.cl.ly/rRukogR5

**Review by premanshup — APPROVED** [review]

**Pull request merged**